src/apps/content-editor/src/app/views/ItemEdit/Content/Actions/Widgets/QuickView/QuickView.js

```diff
@@ -68,7 +68,7 @@ export const QuickView = React.memo(function QuickView(props) {
                 )}
               </span>
             </li>
-            {props.basicApi ? (
+            {codeAccess && (
```

Comment:
Does this short hand `&&` condition have the issue of displaying `0` when the user does not have code acess?

Reply:
No, if there is no access then json link does not render